<?xml version='1.0' encoding='utf-8'?>
<testsuites tests="147" failures="25" errors="0" skipped="45" time="0.0"><testsuite name="SoftwareManagement.Plugin.apama_plugin_install" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_install
08:52:56 INFO  Dir:   plugin_apama/
08:52:56 INFO  Title: Validate apama plugin install
08:52:56 INFO  ==============================================================
08:52:56 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.06 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
<testcase classname="ApamaPluginInstallTest" name="SoftwareManagement.Plugin.apama_plugin_install" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_mon_support" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_mon_support
08:52:56 INFO  Dir:   plugin_apama/
08:52:56 INFO  Title: Validate mon file support of apama plugin
08:52:56 INFO  ==============================================================
08:52:56 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.06 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
<testcase classname="ApamaPluginMonSupportTest" name="SoftwareManagement.Plugin.apama_plugin_mon_support" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_remove
08:52:56 INFO  Dir:   plugin_apama/
08:52:56 INFO  Title: Validate apama plugin remove
08:52:56 INFO  ==============================================================
08:52:56 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.02 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
<testcase classname="ApamaPluginRemoveTest" name="SoftwareManagement.Plugin.apama_plugin_remove" />
</testsuite><testsuite name="SoftwareManagement.Plugin.apama_plugin_update" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<skipped message="SKIPPED: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
08:52:56 INFO  Dir:   plugin_apama/
08:52:56 INFO  Title: Validate apama plugin updating one project with another
08:52:56 INFO  ==============================================================
08:52:56 INFO  Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.02 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Apama plugin tests are disabled by default. Execute `pysys run` with `-XmyPlatform=container` to run these tests
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>09:43:42 INFO  ==============================================================
09:43:42 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
09:43:42 INFO  Dir:   plugin_apama/
09:43:42 INFO  Title: Validate apama plugin updating one project with another
09:43:42 INFO  ==============================================================
09:43:42 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
09:43:43 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
09:43:43 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
09:43:44 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
09:43:45 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
09:43:45 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
09:43:45 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
09:43:46 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
09:43:46 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
09:43:47 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
09:43:47 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

09:43:47 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
09:43:47 WARN  Aborted test due to BLOCKED outcome
09:43:47 INFO  
09:43:47 INFO  cleanup:
09:43:47 INFO  Removing apama project
09:44:03 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
09:44:03 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
09:44:04 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
09:44:04 INFO  
09:44:04 INFO  Test duration: 21.52 secs
09:44:04 INFO  Test final outcome:  BLOCKED
09:44:04 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
09:44:04 INFO  
</system-out>
	</testcase>
<testcase classname="ApamaPluginUpdateTest" name="SoftwareManagement.Plugin.apama_plugin_update">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2" type="BLOCKED" />
		<system-out>09:47:32 INFO  ==============================================================
09:47:32 INFO  Id:    SoftwareManagement.Plugin.apama_plugin_update
09:47:32 INFO  Dir:   plugin_apama/
09:47:32 INFO  Title: Validate apama plugin updating one project with another
09:47:32 INFO  ==============================================================
09:47:32 INFO  Assertion that path exists=False for "/etc/tedge/apama/project" ... passed
09:47:33 INFO  Executed sudo&lt;create_project_dir&gt;, exit status 0
09:47:34 INFO  Executed sudo&lt;unzip_apama_project&gt;, exit status 0
09:47:34 INFO  Executed sudo&lt;move_apama_project&gt;, exit status 0
09:47:35 INFO  Executed sudo&lt;restart_apama_service&gt;, exit status 0
09:47:36 INFO  Executed sudo&lt;apama_correlator_wait&gt;, exit status 0
09:47:36 INFO  Assertion that path exists=True for "/etc/tedge/apama/project" ... passed
09:47:36 INFO  Executed sudo&lt;apama_list_monitors&gt;, exit status 0
09:47:36 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_apama/apama_plugin_update/Output/linux/apama_list_monitors.out ... passed
09:47:37 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
09:47:37 INFO  Contents of plugin_install.err: 
  ERROR: Directory not empty (os error 39)
  -----

09:47:37 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [run.py:35]
09:47:37 WARN  Aborted test due to BLOCKED outcome
09:47:37 INFO  
09:47:37 INFO  cleanup:
09:47:37 INFO  Removing apama project
09:47:53 INFO  Executed sudo&lt;stop_apama_service&gt;, exit status 0, duration 15 secs
09:47:54 INFO  Executed sudo&lt;remove_apama_project&gt;, exit status 0
09:47:54 INFO  Executed sudo&lt;remove_apama_tmp_dir&gt;, exit status 0
09:47:54 INFO  
09:47:54 INFO  Test duration: 21.64 secs
09:47:54 INFO  Test final outcome:  BLOCKED
09:47:54 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2
09:47:54 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_download_path" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:35 INFO  ==============================================================
09:02:35 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
09:02:35 INFO  Dir:   software_management_end_to_end/
09:02:35 INFO  Title: Install package to directory specified in tedge config set tmp.path
09:02:35 INFO  ==============================================================
09:02:35 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:35 WARN  Aborted test due to SKIPPED outcome
09:02:35 INFO  
09:02:35 INFO  Test duration: 0.08 secs
09:02:35 INFO  Test final outcome:  SKIPPED
09:02:35 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:35 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>09:25:38 INFO  ==============================================================
09:25:38 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
09:25:38 INFO  Dir:   software_management_end_to_end/
09:25:38 INFO  Title: Install package to directory specified in tedge config set tmp.path
09:25:38 INFO  ==============================================================
09:25:39 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:25:39 INFO  EnvironmentC8y Setup
09:25:39 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:25:54 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:25:56 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:25:57 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:25:57 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:26:00 INFO  State of current operation: SUCCESSFUL
09:26:00 INFO  Assert that (False == value) with value=False ... passed
09:26:00 INFO  Executed sudo, exit status 0
09:26:01 INFO  Executed sudo, exit status 0
09:26:02 INFO  Executed sudo, exit status 0
09:26:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:26:06 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
09:26:06 INFO  Delaying execution by 8 seconds
09:26:27 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
09:26:28 INFO  Response status: 201
09:26:28 INFO  Response to action: {
    "creationTime": "2022-05-03T09:26:28.201Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298250",
    "id": "18298250",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
09:26:28 INFO  Started operation: {'creationTime': '2022-05-03T09:26:28.201Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298250', 'id': '18298250', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
09:26:28 WARN  , retrying in 0.5 seconds...
09:26:28 WARN  , retrying in 0.5 seconds...
09:26:29 WARN  , retrying in 0.5 seconds...
09:26:29 WARN  , retrying in 0.5 seconds...
09:26:30 WARN  , retrying in 0.5 seconds...
09:26:30 WARN  , retrying in 0.5 seconds...
09:26:31 WARN  , retrying in 0.5 seconds...
09:26:31 WARN  , retrying in 0.5 seconds...
09:26:32 WARN  , retrying in 0.5 seconds...
09:26:32 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
09:26:33 WARN  AssertionError: ... blocked 
09:26:33 INFO  
09:26:33 INFO  cleanup:
09:26:34 INFO  Executed sudo, exit status 0
09:26:34 INFO  Executed sudo, exit status 0
09:26:38 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:26:39 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
09:26:39 INFO  Delaying execution by 8 seconds
09:27:01 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:27:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:27:05 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:27:05 INFO  
09:27:05 INFO  Test duration: 87.22 secs
09:27:05 INFO  Test final outcome:  BLOCKED
09:27:05 INFO  Test outcome reason: AssertionError:
09:27:05 INFO  
</system-out>
	</testcase>
<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED" />
		<system-out>09:47:54 INFO  ==============================================================
09:47:54 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
09:47:54 INFO  Dir:   software_management_end_to_end/
09:47:54 INFO  Title: Install package to directory specified in tedge config set tmp.path
09:47:54 INFO  ==============================================================
09:47:55 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:47:55 INFO  EnvironmentC8y Setup
09:47:56 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:48:10 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:48:11 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:48:11 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:48:12 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:48:14 INFO  State of current operation: FAILED
09:48:14 INFO  Assert that (False == value) with value=False ... passed
09:48:15 INFO  Executed sudo, exit status 0
09:48:15 INFO  Executed sudo, exit status 0
09:48:16 INFO  Executed sudo, exit status 0
09:48:20 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:48:20 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
09:48:20 INFO  Delaying execution by 7 seconds
09:48:41 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
09:48:41 INFO  Response status: 201
09:48:41 INFO  Response to action: {
    "creationTime": "2022-05-03T09:48:41.660Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298368",
    "id": "18298368",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "::apt",
            "url": "https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463"
        }
    ]
}
09:48:41 INFO  Started operation: {'creationTime': '2022-05-03T09:48:41.660Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298368', 'id': '18298368', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]", 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
09:48:41 WARN  , retrying in 0.5 seconds...
09:48:42 WARN  , retrying in 0.5 seconds...
09:48:42 WARN  , retrying in 0.5 seconds...
09:48:43 WARN  , retrying in 0.5 seconds...
09:48:43 WARN  , retrying in 0.5 seconds...
09:48:44 WARN  , retrying in 0.5 seconds...
09:48:44 WARN  , retrying in 0.5 seconds...
09:48:45 WARN  , retrying in 0.5 seconds...
09:48:45 WARN  , retrying in 0.5 seconds...
09:48:46 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 74, in retry_decorator
    logger)
  File "/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py", line 33, in __retry_internal
    return f()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py", line 28, in assert_install_in_download_path
    assert "rolldice" in os.listdir(f"{install_directory}")
AssertionError
09:48:46 WARN  AssertionError: ... blocked 
09:48:46 INFO  
09:48:46 INFO  cleanup:
09:48:46 INFO  Executed sudo, exit status 0
09:48:47 INFO  Executed sudo, exit status 0
09:48:51 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:48:51 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
09:48:51 INFO  Delaying execution by 7 seconds
09:49:12 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:49:17 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:49:17 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:49:17 INFO  
09:49:17 INFO  Test duration: 83.04 secs
09:49:17 INFO  Test final outcome:  BLOCKED
09:49:17 INFO  Test outcome reason: AssertionError:
09:49:17 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_fail" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:35 INFO  ==============================================================
09:02:35 INFO  Id:    SoftwareManagement.sm_apt_install_fail
09:02:35 INFO  Dir:   software_management_end_to_end/
09:02:35 INFO  Title: Validate end to end behaviour for a failing installation
09:02:35 INFO  ==============================================================
09:02:35 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:35 WARN  Aborted test due to SKIPPED outcome
09:02:35 INFO  
09:02:35 INFO  Test duration: 0.02 secs
09:02:35 INFO  Test final outcome:  SKIPPED
09:02:35 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:35 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_fail" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:35 INFO  ==============================================================
09:02:35 INFO  Id:    SoftwareManagement.sm_apt_install_remove
09:02:35 INFO  Dir:   software_management_end_to_end/
09:02:35 INFO  Title: Validate end to end behaviour for the apt plugin for installation and removal of packages
09:02:35 INFO  ==============================================================
09:02:35 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:35 WARN  Aborted test due to SKIPPED outcome
09:02:35 INFO  
09:02:35 INFO  Test duration: 0.02 secs
09:02:35 INFO  Test final outcome:  SKIPPED
09:02:35 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:35 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_default" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:35 INFO  ==============================================================
09:02:35 INFO  Id:    SoftwareManagement.sm_apt_install_remove_default
09:02:35 INFO  Dir:   software_management_end_to_end/
09:02:35 INFO  Title: Validate end to end behaviour for the default plugin for installation and removal of packages
09:02:35 INFO  ==============================================================
09:02:35 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:35 WARN  Aborted test due to SKIPPED outcome
09:02:35 INFO  
09:02:35 INFO  Test duration: 0.02 secs
09:02:35 INFO  Test final outcome:  SKIPPED
09:02:35 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:35 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_default" />
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:35 INFO  ==============================================================
09:02:35 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
09:02:35 INFO  Dir:   software_management_end_to_end/
09:02:35 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
09:02:35 INFO  ==============================================================
09:02:35 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:35 WARN  Aborted test due to SKIPPED outcome
09:02:35 INFO  
09:02:35 INFO  Test duration: 0.02 secs
09:02:35 INFO  Test final outcome:  SKIPPED
09:02:35 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:35 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True" type="FAILED" />
		<system-out>09:33:52 INFO  ==============================================================
09:33:52 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
09:33:52 INFO  Dir:   software_management_end_to_end/
09:33:52 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
09:33:52 INFO  ==============================================================
09:33:52 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:33:52 INFO  EnvironmentC8y Setup
09:33:53 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:33:53 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
09:33:53 INFO  Delaying execution by 7 seconds
09:34:14 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:34:15 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:34:15 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:34:16 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:34:18 INFO  State of current operation: SUCCESSFUL
09:34:18 INFO  Assert that (False == value) with value=False ... passed
09:34:18 INFO  Assert that (False == value) with value=False ... passed
09:34:19 INFO  Assert that (False == value) with value=False ... passed
09:34:19 INFO  Package robotfindskitten is installed
09:34:19 WARN  Assert that (False == value) with value=True ... failed 
09:34:19 INFO  Response status: 201
09:34:19 INFO  Response to action: {
    "creationTime": "2022-05-03T09:34:19.535Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298305",
    "id": "18298305",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:34:19 INFO  Started operation: {'creationTime': '2022-05-03T09:34:19.535Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298305', 'id': '18298305', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
09:34:21 INFO  State of operation 18298305 : PENDING
09:34:24 INFO  State of operation 18298305 : EXECUTING
09:34:26 INFO  State of operation 18298305 : EXECUTING
09:34:28 INFO  State of operation 18298305 : EXECUTING
09:34:30 INFO  State of operation 18298305 : EXECUTING
09:34:32 INFO  State of operation 18298305 : EXECUTING
09:34:35 INFO  State of operation 18298305 : EXECUTING
09:34:38 INFO  State of operation 18298305 : EXECUTING
09:34:40 INFO  State of operation 18298305 : EXECUTING
09:34:43 INFO  State of operation 18298305 : EXECUTING
09:34:45 INFO  State of operation 18298305 : EXECUTING
09:34:47 INFO  State of operation 18298305 : EXECUTING
09:34:49 INFO  State of operation 18298305 : EXECUTING
09:34:52 INFO  State of operation 18298305 : EXECUTING
09:34:54 INFO  State of operation 18298305 : EXECUTING
09:34:56 INFO  State of operation 18298305 : EXECUTING
09:34:58 INFO  State of operation 18298305 : EXECUTING
09:35:01 INFO  State of operation 18298305 : EXECUTING
09:35:03 INFO  State of operation 18298305 : EXECUTING
09:35:06 INFO  State of operation 18298305 : EXECUTING
09:35:08 INFO  State of operation 18298305 : EXECUTING
09:35:11 INFO  State of operation 18298305 : EXECUTING
09:35:13 INFO  State of operation 18298305 : EXECUTING
09:35:15 INFO  State of operation 18298305 : EXECUTING
09:35:17 INFO  State of operation 18298305 : EXECUTING
09:35:19 INFO  State of operation 18298305 : EXECUTING
09:35:22 INFO  State of operation 18298305 : EXECUTING
09:35:24 INFO  State of operation 18298305 : EXECUTING
09:35:27 INFO  State of operation 18298305 : EXECUTING
09:35:29 INFO  State of operation 18298305 : EXECUTING
09:35:31 INFO  State of operation 18298305 : EXECUTING
09:35:34 INFO  State of operation 18298305 : EXECUTING
09:35:36 INFO  State of operation 18298305 : EXECUTING
09:35:38 INFO  State of operation 18298305 : EXECUTING
09:35:40 INFO  State of operation 18298305 : SUCCESSFUL
09:35:40 INFO  Assert that (False == value) with value=False ... passed
09:35:41 INFO  Package asciijump is installed
09:35:41 INFO  Assert that (True == value) with value=True ... passed
09:35:41 INFO  Package squirrel3 is installed
09:35:41 INFO  Assert that (True == value) with value=True ... passed
09:35:41 INFO  Package robotfindskitten is installed
09:35:41 INFO  Assert that (True == value) with value=True ... passed
09:35:41 INFO  Response status: 201
09:35:41 INFO  Response to action: {
    "creationTime": "2022-05-03T09:35:41.854Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298315",
    "id": "18298315",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:35:41 INFO  Started operation: {'creationTime': '2022-05-03T09:35:41.854Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298315', 'id': '18298315', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
09:35:44 INFO  State of operation 18298315 : PENDING
09:35:46 INFO  State of operation 18298315 : EXECUTING
09:35:48 INFO  State of operation 18298315 : EXECUTING
09:35:50 INFO  State of operation 18298315 : EXECUTING
09:35:53 INFO  State of operation 18298315 : EXECUTING
09:35:55 INFO  State of operation 18298315 : EXECUTING
09:35:58 INFO  State of operation 18298315 : EXECUTING
09:36:01 INFO  State of operation 18298315 : EXECUTING
09:36:03 INFO  State of operation 18298315 : EXECUTING
09:36:05 INFO  State of operation 18298315 : EXECUTING
09:36:07 INFO  State of operation 18298315 : EXECUTING
09:36:10 INFO  State of operation 18298315 : EXECUTING
09:36:12 INFO  State of operation 18298315 : EXECUTING
09:36:14 INFO  State of operation 18298315 : EXECUTING
09:36:16 INFO  State of operation 18298315 : EXECUTING
09:36:19 INFO  State of operation 18298315 : EXECUTING
09:36:21 INFO  State of operation 18298315 : EXECUTING
09:36:24 INFO  State of operation 18298315 : EXECUTING
09:36:26 INFO  State of operation 18298315 : EXECUTING
09:36:28 INFO  State of operation 18298315 : EXECUTING
09:36:31 INFO  State of operation 18298315 : EXECUTING
09:36:33 INFO  State of operation 18298315 : EXECUTING
09:36:35 INFO  State of operation 18298315 : EXECUTING
09:36:37 INFO  State of operation 18298315 : EXECUTING
09:36:40 INFO  State of operation 18298315 : EXECUTING
09:36:42 INFO  State of operation 18298315 : EXECUTING
09:36:45 INFO  State of operation 18298315 : EXECUTING
09:36:47 INFO  State of operation 18298315 : SUCCESSFUL
09:36:47 INFO  Assert that (False == value) with value=False ... passed
09:36:47 INFO  Assert that (False == value) with value=False ... passed
09:36:48 INFO  Assert that (False == value) with value=False ... passed
09:36:48 INFO  Assert that (False == value) with value=False ... passed
09:36:48 INFO  
09:36:48 INFO  cleanup:
09:36:52 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:36:52 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:36:52 INFO  
09:36:52 INFO  Test duration: 180.12 secs
09:36:52 INFO  Test final outcome:  FAILED
09:36:52 INFO  Test outcome reason: Assert that (False == value) with value=True
09:36:52 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple">
		<failure message="FAILED: Assert that (False == value) with value=True (+2 other failures)" type="FAILED" />
		<system-out>09:55:40 INFO  ==============================================================
09:55:40 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple
09:55:40 INFO  Dir:   software_management_end_to_end/
09:55:40 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages
09:55:40 INFO  ==============================================================
09:55:40 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:55:40 INFO  EnvironmentC8y Setup
09:55:41 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:55:41 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
09:55:41 INFO  Delaying execution by 7 seconds
09:56:02 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:56:03 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:56:03 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:56:04 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:56:06 INFO  State of current operation: SUCCESSFUL
09:56:06 INFO  Assert that (False == value) with value=False ... passed
09:56:06 INFO  Package asciijump is installed
09:56:07 WARN  Assert that (False == value) with value=True ... failed 
09:56:07 INFO  Package squirrel3 is installed
09:56:07 WARN  Assert that (False == value) with value=True ... failed 
09:56:07 INFO  Package robotfindskitten is installed
09:56:07 WARN  Assert that (False == value) with value=True ... failed 
09:56:07 INFO  Response status: 201
09:56:07 INFO  Response to action: {
    "creationTime": "2022-05-03T09:56:07.839Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298441",
    "id": "18298441",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:56:07 INFO  Started operation: {'creationTime': '2022-05-03T09:56:07.839Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298441', 'id': '18298441', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
09:56:10 INFO  State of operation 18298441 : PENDING
09:56:12 INFO  State of operation 18298441 : EXECUTING
09:56:14 INFO  State of operation 18298441 : EXECUTING
09:56:16 INFO  State of operation 18298441 : EXECUTING
09:56:19 INFO  State of operation 18298441 : EXECUTING
09:56:21 INFO  State of operation 18298441 : EXECUTING
09:56:23 INFO  State of operation 18298441 : EXECUTING
09:56:26 INFO  State of operation 18298441 : EXECUTING
09:56:29 INFO  State of operation 18298441 : EXECUTING
09:56:31 INFO  State of operation 18298441 : EXECUTING
09:56:33 INFO  State of operation 18298441 : EXECUTING
09:56:36 INFO  State of operation 18298441 : EXECUTING
09:56:38 INFO  State of operation 18298441 : EXECUTING
09:56:40 INFO  State of operation 18298441 : EXECUTING
09:56:42 INFO  State of operation 18298441 : EXECUTING
09:56:45 INFO  State of operation 18298441 : SUCCESSFUL
09:56:45 INFO  Assert that (False == value) with value=False ... passed
09:56:45 INFO  Package asciijump is installed
09:56:45 INFO  Assert that (True == value) with value=True ... passed
09:56:45 INFO  Package squirrel3 is installed
09:56:45 INFO  Assert that (True == value) with value=True ... passed
09:56:46 INFO  Package robotfindskitten is installed
09:56:46 INFO  Assert that (True == value) with value=True ... passed
09:56:46 INFO  Response status: 201
09:56:46 INFO  Response to action: {
    "creationTime": "2022-05-03T09:56:46.225Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298447",
    "id": "18298447",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "delete",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "delete",
            "id": "5474869",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "delete",
            "id": "5475279",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:56:46 INFO  Started operation: {'creationTime': '2022-05-03T09:56:46.225Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298447', 'id': '18298447', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'delete', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': '::apt'}, {'action': 'delete', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'delete', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'delete', 'id': '5474869', 'version': '::apt', 'url': ' '}, {'name': 'squirrel3', 'action': 'delete', 'id': '5475279', 'version': '::apt', 'url': ' '}]}
09:56:48 INFO  State of operation 18298447 : PENDING
09:56:50 INFO  State of operation 18298447 : EXECUTING
09:56:52 INFO  State of operation 18298447 : EXECUTING
09:56:55 INFO  State of operation 18298447 : EXECUTING
09:56:57 INFO  State of operation 18298447 : EXECUTING
09:56:59 INFO  State of operation 18298447 : EXECUTING
09:57:02 INFO  State of operation 18298447 : EXECUTING
09:57:04 INFO  State of operation 18298447 : EXECUTING
09:57:07 INFO  State of operation 18298447 : EXECUTING
09:57:09 INFO  State of operation 18298447 : EXECUTING
09:57:11 INFO  State of operation 18298447 : EXECUTING
09:57:14 INFO  State of operation 18298447 : EXECUTING
09:57:16 INFO  State of operation 18298447 : EXECUTING
09:57:18 INFO  State of operation 18298447 : EXECUTING
09:57:20 INFO  State of operation 18298447 : EXECUTING
09:57:23 INFO  State of operation 18298447 : EXECUTING
09:57:25 INFO  State of operation 18298447 : EXECUTING
09:57:27 INFO  State of operation 18298447 : EXECUTING
09:57:30 INFO  State of operation 18298447 : EXECUTING
09:57:32 INFO  State of operation 18298447 : EXECUTING
09:57:34 INFO  State of operation 18298447 : EXECUTING
09:57:37 INFO  State of operation 18298447 : EXECUTING
09:57:39 INFO  State of operation 18298447 : EXECUTING
09:57:41 INFO  State of operation 18298447 : EXECUTING
09:57:44 INFO  State of operation 18298447 : EXECUTING
09:57:46 INFO  State of operation 18298447 : EXECUTING
09:57:48 INFO  State of operation 18298447 : EXECUTING
09:57:51 INFO  State of operation 18298447 : EXECUTING
09:57:53 INFO  State of operation 18298447 : SUCCESSFUL
09:57:53 INFO  Assert that (False == value) with value=False ... passed
09:57:53 INFO  Assert that (False == value) with value=False ... passed
09:57:54 INFO  Assert that (False == value) with value=False ... passed
09:57:54 INFO  Assert that (False == value) with value=False ... passed
09:57:54 INFO  
09:57:54 INFO  cleanup:
09:57:58 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:57:59 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:57:59 INFO  
09:57:59 INFO  Test duration: 138.34 secs
09:57:59 INFO  Test final outcome:  FAILED
09:57:59 INFO  Test outcome reason: Assert that (False == value) with value=True (+2 other failures)
09:57:59 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_multiple_mixed" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.04 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False" type="BLOCKED" />
		<system-out>09:36:53 INFO  ==============================================================
09:36:53 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
09:36:53 INFO  Dir:   software_management_end_to_end/
09:36:53 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
09:36:53 INFO  ==============================================================
09:36:53 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:36:53 INFO  EnvironmentC8y Setup
09:36:53 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:36:53 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
09:36:53 INFO  Delaying execution by 6 seconds
09:37:13 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:37:14 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:37:14 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:37:15 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:37:17 INFO  State of current operation: SUCCESSFUL
09:37:17 INFO  Assert that (False == value) with value=False ... passed
09:37:18 INFO  Assert that (False == value) with value=False ... passed
09:37:18 INFO  Assert that (False == value) with value=False ... passed
09:37:18 INFO  Assert that (False == value) with value=False ... passed
09:37:18 INFO  Assert that (False == value) with value=False ... passed
09:37:19 INFO  Response status: 201
09:37:19 INFO  Response to action: {
    "creationTime": "2022-05-03T09:37:19.957Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298325",
    "id": "18298325",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:37:19 INFO  Started operation: {'creationTime': '2022-05-03T09:37:19.957Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298325', 'id': '18298325', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
09:37:22 INFO  State of operation 18298325 : PENDING
09:37:24 INFO  State of operation 18298325 : EXECUTING
09:37:26 INFO  State of operation 18298325 : EXECUTING
09:37:28 INFO  State of operation 18298325 : EXECUTING
09:37:31 INFO  State of operation 18298325 : EXECUTING
09:37:33 INFO  State of operation 18298325 : EXECUTING
09:37:36 INFO  State of operation 18298325 : EXECUTING
09:37:38 INFO  State of operation 18298325 : EXECUTING
09:37:41 INFO  State of operation 18298325 : EXECUTING
09:37:43 INFO  State of operation 18298325 : EXECUTING
09:37:45 INFO  State of operation 18298325 : EXECUTING
09:37:48 INFO  State of operation 18298325 : EXECUTING
09:37:50 INFO  State of operation 18298325 : EXECUTING
09:37:52 INFO  State of operation 18298325 : EXECUTING
09:37:54 INFO  State of operation 18298325 : EXECUTING
09:37:57 INFO  State of operation 18298325 : EXECUTING
09:37:59 INFO  State of operation 18298325 : EXECUTING
09:38:01 INFO  State of operation 18298325 : EXECUTING
09:38:04 INFO  State of operation 18298325 : EXECUTING
09:38:06 INFO  State of operation 18298325 : EXECUTING
09:38:09 INFO  State of operation 18298325 : EXECUTING
09:38:11 INFO  State of operation 18298325 : EXECUTING
09:38:14 INFO  State of operation 18298325 : EXECUTING
09:38:16 INFO  State of operation 18298325 : EXECUTING
09:38:19 INFO  State of operation 18298325 : EXECUTING
09:38:21 INFO  State of operation 18298325 : EXECUTING
09:38:23 INFO  State of operation 18298325 : EXECUTING
09:38:25 INFO  State of operation 18298325 : EXECUTING
09:38:27 INFO  State of operation 18298325 : EXECUTING
09:38:30 INFO  State of operation 18298325 : EXECUTING
09:38:32 INFO  State of operation 18298325 : EXECUTING
09:38:34 INFO  State of operation 18298325 : EXECUTING
09:38:36 INFO  State of operation 18298325 : EXECUTING
09:38:39 INFO  State of operation 18298325 : EXECUTING
09:38:41 INFO  State of operation 18298325 : EXECUTING
09:38:43 INFO  State of operation 18298325 : EXECUTING
09:38:45 INFO  State of operation 18298325 : EXECUTING
09:38:48 INFO  State of operation 18298325 : EXECUTING
09:38:50 INFO  State of operation 18298325 : EXECUTING
09:38:53 INFO  State of operation 18298325 : EXECUTING
09:38:55 INFO  State of operation 18298325 : EXECUTING
09:38:58 INFO  State of operation 18298325 : EXECUTING
09:39:00 INFO  State of operation 18298325 : EXECUTING
09:39:02 INFO  State of operation 18298325 : EXECUTING
09:39:04 INFO  State of operation 18298325 : EXECUTING
09:39:07 INFO  State of operation 18298325 : EXECUTING
09:39:09 INFO  State of operation 18298325 : EXECUTING
09:39:11 INFO  State of operation 18298325 : EXECUTING
09:39:14 INFO  State of operation 18298325 : EXECUTING
09:39:16 INFO  State of operation 18298325 : EXECUTING
09:39:18 INFO  State of operation 18298325 : EXECUTING
09:39:21 INFO  State of operation 18298325 : EXECUTING
09:39:23 INFO  State of operation 18298325 : EXECUTING
09:39:25 INFO  State of operation 18298325 : EXECUTING
09:39:28 INFO  State of operation 18298325 : EXECUTING
09:39:30 INFO  State of operation 18298325 : EXECUTING
09:39:33 INFO  State of operation 18298325 : EXECUTING
09:39:35 INFO  State of operation 18298325 : EXECUTING
09:39:37 INFO  State of operation 18298325 : EXECUTING
09:39:40 INFO  State of operation 18298325 : EXECUTING
09:39:42 INFO  State of operation 18298325 : EXECUTING
09:39:44 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
09:39:44 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
09:39:44 INFO  
09:39:44 INFO  cleanup:
09:39:51 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:39:52 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:39:52 INFO  
09:39:52 INFO  Test duration: 179.27 secs
09:39:52 INFO  Test final outcome:  BLOCKED
09:39:52 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False
09:39:52 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_multiple_mixed">
		<failure message="BLOCKED: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)" type="BLOCKED" />
		<system-out>09:57:59 INFO  ==============================================================
09:57:59 INFO  Id:    SoftwareManagement.sm_apt_install_remove_multiple_mixed
09:57:59 INFO  Dir:   software_management_end_to_end/
09:57:59 INFO  Title: Validate end to end behaviour for the apt plugin for multiple packages with mixed versions
09:57:59 INFO  ==============================================================
09:57:59 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:57:59 INFO  EnvironmentC8y Setup
09:57:59 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:57:59 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
09:57:59 INFO  Delaying execution by 6 seconds
09:58:19 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
09:58:20 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:58:20 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:58:21 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:58:23 INFO  State of current operation: SUCCESSFUL
09:58:23 INFO  Assert that (False == value) with value=False ... passed
09:58:23 INFO  Assert that (False == value) with value=False ... passed
09:58:23 INFO  Assert that (False == value) with value=False ... passed
09:58:24 INFO  Assert that (False == value) with value=False ... passed
09:58:24 INFO  Package moon-buggy is installed
09:58:24 WARN  Assert that (False == value) with value=True ... failed 
09:58:25 INFO  Response status: 201
09:58:25 INFO  Response to action: {
    "creationTime": "2022-05-03T09:58:25.616Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298456",
    "id": "18298456",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "robotfindskitten",
            "action": "install",
            "id": "5474869",
            "version": " ",
            "url": " "
        },
        {
            "name": "squirrel3",
            "action": "install",
            "id": "5475279",
            "version": "3.1-7::apt",
            "url": " "
        },
        {
            "name": "rolldice",
            "action": "install",
            "id": "5152439",
            "version": "1.16-1+b1",
            "url": " "
        },
        {
            "name": "moon-buggy",
            "action": "install",
            "id": "5439204",
            "version": "1:1.0.51-12",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        }
    ]
}
09:58:25 INFO  Started operation: {'creationTime': '2022-05-03T09:58:25.616Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298456', 'id': '18298456', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5474869', 'name': 'robotfindskitten', 'url': ' ', 'version': ' '}, {'action': 'install', 'id': '5475279', 'name': 'squirrel3', 'url': ' ', 'version': '3.1-7::apt'}, {'action': 'install', 'id': '5152439', 'name': 'rolldice', 'url': ' ', 'version': '1.16-1+b1'}, {'action': 'install', 'id': '5439204', 'name': 'moon-buggy', 'url': ' ', 'version': '1:1.0.51-12'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}]", 'c8y_SoftwareUpdate': [{'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'robotfindskitten', 'action': 'install', 'id': '5474869', 'version': ' ', 'url': ' '}, {'name': 'squirrel3', 'action': 'install', 'id': '5475279', 'version': '3.1-7::apt', 'url': ' '}, {'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '1.16-1+b1', 'url': ' '}, {'name': 'moon-buggy', 'action': 'install', 'id': '5439204', 'version': '1:1.0.51-12', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}]}
09:58:27 INFO  State of operation 18298456 : PENDING
09:58:30 INFO  State of operation 18298456 : EXECUTING
09:58:32 INFO  State of operation 18298456 : EXECUTING
09:58:34 INFO  State of operation 18298456 : EXECUTING
09:58:36 INFO  State of operation 18298456 : EXECUTING
09:58:38 INFO  State of operation 18298456 : EXECUTING
09:58:41 INFO  State of operation 18298456 : EXECUTING
09:58:44 INFO  State of operation 18298456 : EXECUTING
09:58:46 INFO  State of operation 18298456 : EXECUTING
09:58:48 INFO  State of operation 18298456 : EXECUTING
09:58:51 INFO  State of operation 18298456 : EXECUTING
09:58:53 INFO  State of operation 18298456 : EXECUTING
09:58:55 INFO  State of operation 18298456 : EXECUTING
09:58:57 INFO  State of operation 18298456 : EXECUTING
09:59:00 INFO  State of operation 18298456 : EXECUTING
09:59:02 INFO  State of operation 18298456 : EXECUTING
09:59:04 INFO  State of operation 18298456 : EXECUTING
09:59:07 INFO  State of operation 18298456 : EXECUTING
09:59:09 INFO  State of operation 18298456 : EXECUTING
09:59:11 INFO  State of operation 18298456 : EXECUTING
09:59:14 INFO  State of operation 18298456 : EXECUTING
09:59:16 INFO  State of operation 18298456 : EXECUTING
09:59:19 INFO  State of operation 18298456 : EXECUTING
09:59:21 INFO  State of operation 18298456 : EXECUTING
09:59:23 INFO  State of operation 18298456 : EXECUTING
09:59:26 INFO  State of operation 18298456 : EXECUTING
09:59:28 INFO  State of operation 18298456 : EXECUTING
09:59:31 INFO  State of operation 18298456 : EXECUTING
09:59:33 INFO  State of operation 18298456 : EXECUTING
09:59:35 INFO  State of operation 18298456 : EXECUTING
09:59:38 INFO  State of operation 18298456 : EXECUTING
09:59:40 INFO  State of operation 18298456 : EXECUTING
09:59:42 INFO  State of operation 18298456 : EXECUTING
09:59:44 INFO  State of operation 18298456 : EXECUTING
09:59:47 INFO  State of operation 18298456 : EXECUTING
09:59:49 INFO  State of operation 18298456 : EXECUTING
09:59:51 INFO  State of operation 18298456 : EXECUTING
09:59:53 INFO  State of operation 18298456 : EXECUTING
09:59:56 INFO  State of operation 18298456 : EXECUTING
09:59:59 INFO  State of operation 18298456 : EXECUTING
10:00:01 INFO  State of operation 18298456 : EXECUTING
10:00:03 INFO  State of operation 18298456 : EXECUTING
10:00:06 INFO  State of operation 18298456 : EXECUTING
10:00:08 INFO  State of operation 18298456 : EXECUTING
10:00:10 INFO  State of operation 18298456 : EXECUTING
10:00:12 INFO  State of operation 18298456 : EXECUTING
10:00:15 INFO  State of operation 18298456 : EXECUTING
10:00:17 INFO  State of operation 18298456 : EXECUTING
10:00:19 INFO  State of operation 18298456 : EXECUTING
10:00:22 INFO  State of operation 18298456 : EXECUTING
10:00:24 INFO  State of operation 18298456 : EXECUTING
10:00:26 INFO  State of operation 18298456 : EXECUTING
10:00:29 INFO  State of operation 18298456 : EXECUTING
10:00:31 INFO  State of operation 18298456 : EXECUTING
10:00:33 INFO  State of operation 18298456 : EXECUTING
10:00:36 INFO  State of operation 18298456 : EXECUTING
10:00:38 INFO  State of operation 18298456 : EXECUTING
10:00:40 INFO  State of operation 18298456 : EXECUTING
10:00:43 INFO  State of operation 18298456 : EXECUTING
10:00:45 INFO  State of operation 18298456 : EXECUTING
10:00:47 INFO  State of operation 18298456 : EXECUTING
10:00:49 WARN  SystemError occurred while running test - Timeout while waiting for status SUCCESSFUL or False
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_remove_multiple_mixed/run.py", line 89, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 381, in wait_until_status
    "Timeout while waiting for status %s or %s" % (status, status2)
SystemError: Timeout while waiting for status SUCCESSFUL or False
10:00:49 WARN  SystemError: Timeout while waiting for status SUCCESSFUL or False ... blocked 
10:00:49 INFO  
10:00:49 INFO  cleanup:
10:00:55 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:00:56 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:00:56 INFO  
10:00:56 INFO  Test duration: 177.13 secs
10:00:56 INFO  Test final outcome:  BLOCKED
10:00:56 INFO  Test outcome reason: SystemError: Timeout while waiting for status SUCCESSFUL or False (+1 other failures)
10:00:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_apt_install_remove_with_version" tests="3" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_apt_install_remove_with_version
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the apt plugin for packages with version
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.02 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
<testcase classname="PySysTest" name="SoftwareManagement.sm_apt_install_remove_with_version" />
</testsuite><testsuite name="SoftwareManagement.sm_docker_install_remove_multiple" tests="3" errors="0" failures="2" skipped="1" time="0"><testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.02 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.Use parameter -XmyPlatform='container' to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>09:45:07 INFO  ==============================================================
09:45:07 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
09:45:07 INFO  Dir:   software_management_end_to_end/
09:45:07 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
09:45:07 INFO  ==============================================================
09:45:08 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
09:45:08 INFO  EnvironmentC8y Setup
09:45:09 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
09:45:23 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
09:45:24 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
09:45:24 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
09:45:24 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
09:45:27 INFO  State of current operation: SUCCESSFUL
09:45:27 INFO  Response status: 201
09:45:27 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-03T09:45:27.409Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-03T09:45:27.395Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298354",
    "id": "18298354",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
09:45:27 INFO  Started operation: {'delivery': {'time': '2022-05-03T09:45:27.409Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-03T09:45:27.395Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298354', 'id': '18298354', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
09:45:29 INFO  State of operation 18298354 : PENDING
09:45:31 INFO  State of operation 18298354 : EXECUTING
09:45:34 INFO  State of operation 18298354 : EXECUTING
09:45:36 INFO  State of operation 18298354 : EXECUTING
09:45:38 INFO  State of operation 18298354 : EXECUTING
09:45:40 INFO  State of operation 18298354 : FAILED
09:45:40 ERROR Stopping as the operation has failed
09:45:40 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
09:45:40 WARN  SystemError: The operation has failed ... blocked 
09:45:40 INFO  
09:45:40 INFO  cleanup:
09:45:44 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
09:45:45 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
09:45:45 INFO  
09:45:45 INFO  Test duration: 38.06 secs
09:45:45 INFO  Test final outcome:  BLOCKED
09:45:45 INFO  Test outcome reason: SystemError: The operation has failed
09:45:45 INFO  
</system-out>
	</testcase>
<testcase classname="SMDockerInstallRemove" name="SoftwareManagement.sm_docker_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed" type="BLOCKED" />
		<system-out>10:03:32 INFO  ==============================================================
10:03:32 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
10:03:32 INFO  Dir:   software_management_end_to_end/
10:03:32 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
10:03:32 INFO  ==============================================================
10:03:32 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
10:03:32 INFO  EnvironmentC8y Setup
10:03:33 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:03:33 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:03:33 INFO  Delaying execution by 6 seconds
10:03:53 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
10:03:54 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:03:54 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:03:55 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:03:57 INFO  State of current operation: SUCCESSFUL
10:03:57 INFO  Response status: 201
10:03:57 INFO  Response to action: {
    "creationTime": "2022-05-03T10:03:57.674Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298480",
    "id": "18298480",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
10:03:57 INFO  Started operation: {'creationTime': '2022-05-03T10:03:57.674Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298480', 'id': '18298480', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
10:03:59 INFO  State of operation 18298480 : PENDING
10:04:02 INFO  State of operation 18298480 : EXECUTING
10:04:04 INFO  State of operation 18298480 : EXECUTING
10:04:06 INFO  State of operation 18298480 : EXECUTING
10:04:08 INFO  State of operation 18298480 : FAILED
10:04:08 ERROR Stopping as the operation has failed
10:04:08 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
10:04:08 WARN  SystemError: The operation has failed ... blocked 
10:04:08 INFO  
10:04:08 INFO  cleanup:
10:04:12 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:04:13 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:04:13 INFO  
10:04:13 INFO  Test duration: 40.58 secs
10:04:13 INFO  Test final outcome:  BLOCKED
10:04:13 INFO  Test outcome reason: SystemError: The operation has failed
10:04:13 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_fake_install_remove
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the dummy plugin for installation and removal of packages
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.02 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove" />
</testsuite><testsuite name="SoftwareManagement.sm_fake_install_remove_multiple" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_fake_install_remove_multiple
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.02 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_fake_install_remove_multiple" />
</testsuite><testsuite name="SoftwareManagement.sm_mapper_fail_and_reconnect" tests="2" errors="0" failures="0" skipped="2" time="0"><testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
09:02:36 INFO  ==============================================================
09:02:36 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.02 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="MapperC8yReceiveLastMessageOnRestart" name="SoftwareManagement.sm_mapper_fail_and_reconnect">
		<skipped message="SKIPPED: instable https://github.com/thin-edge/thin-edge.io/issues/752" />
		<system-out>10:05:47 INFO  ==============================================================
10:05:47 INFO  Id:    SoftwareManagement.sm_mapper_fail_and_reconnect
10:05:47 INFO  Dir:   software_management_end_to_end/
10:05:47 INFO  Title: Validate the tedge-mapper-sm-c8y for receiving the last message on restart
10:05:47 INFO  ==============================================================
10:05:47 INFO  instable https://github.com/thin-edge/thin-edge.io/issues/752 ... skipped
10:05:47 INFO  
10:05:47 INFO  Test duration: 0.07 secs
10:05:47 INFO  Test final outcome:  SKIPPED
10:05:47 INFO  Test outcome reason: instable https://github.com/thin-edge/thin-edge.io/issues/752
10:05:47 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="SoftwareManagement.sm_mixed_install_remove_multiple" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<skipped message="SKIPPED: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it" />
		<system-out>09:02:36 INFO  ==============================================================
09:02:36 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
09:02:36 INFO  Dir:   software_management_end_to_end/
09:02:36 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
09:02:36 INFO  ==============================================================
09:02:36 INFO  Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it ... skipped
09:02:36 WARN  Aborted test due to SKIPPED outcome
09:02:36 INFO  
09:02:36 INFO  Test duration: 0.03 secs
09:02:36 INFO  Test final outcome:  SKIPPED
09:02:36 INFO  Test outcome reason: Testing the fake plugin is not enabled.Use parameter -Xfakeplugin=fakeplugin to enable it
09:02:36 INFO  
</system-out>
	</testcase>
<testcase classname="PySysTest" name="SoftwareManagement.sm_mixed_install_remove_multiple">
		<failure message="BLOCKED: SystemError: The operation has failed (+1 other failures)" type="BLOCKED" />
		<system-out>10:05:47 INFO  ==============================================================
10:05:47 INFO  Id:    SoftwareManagement.sm_mixed_install_remove_multiple
10:05:47 INFO  Dir:   software_management_end_to_end/
10:05:47 INFO  Title: Validate end to end behaviour for the dummy-plugin for multiple packages with mixed versions
10:05:47 INFO  ==============================================================
10:05:47 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
10:05:47 INFO  EnvironmentC8y Setup
10:05:47 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
10:05:47 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
10:05:47 INFO  Delaying execution by 6 seconds
10:06:07 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
10:06:08 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
10:06:08 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
10:06:09 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
10:06:11 INFO  State of current operation: SUCCESSFUL
10:06:11 INFO  Package apple is installed
10:06:11 INFO  Assert that (True == value) with value=True ... passed
10:06:11 INFO  Package banana is installed
10:06:11 INFO  Assert that (True == value) with value=True ... passed
10:06:12 INFO  Package cherry is installed
10:06:12 INFO  Assert that (True == value) with value=True ... passed
10:06:12 INFO  Package asciijump is installed
10:06:12 WARN  Assert that (False == value) with value=True ... failed 
10:06:12 INFO  Assert that (False == value) with value=False ... passed
10:06:12 INFO  Response status: 201
10:06:12 INFO  Response to action: {
    "delivery": {
        "time": "2022-05-03T10:06:12.816Z",
        "log": [],
        "status": "PENDING"
    },
    "creationTime": "2022-05-03T10:06:12.804Z",
    "deviceName": "m32sd10b",
    "deviceId": "18269384",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298537",
    "id": "18298537",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "apple",
            "action": "install",
            "id": "5495053",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "banana",
            "action": "install",
            "id": "5494888",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "asciijump",
            "action": "install",
            "id": "5475369",
            "version": "::apt",
            "url": " "
        },
        {
            "name": "cherry",
            "action": "install",
            "id": "5495382",
            "version": "::fruits",
            "url": " "
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "::docker",
            "url": " "
        }
    ]
}
10:06:12 INFO  Started operation: {'delivery': {'time': '2022-05-03T10:06:12.816Z', 'log': [], 'status': 'PENDING'}, 'creationTime': '2022-05-03T10:06:12.804Z', 'deviceName': 'm32sd10b', 'deviceId': '18269384', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/18298537', 'id': '18298537', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '5495053', 'name': 'apple', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5494888', 'name': 'banana', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '5475369', 'name': 'asciijump', 'url': ' ', 'version': '::apt'}, {'action': 'install', 'id': '5495382', 'name': 'cherry', 'url': ' ', 'version': '::fruits'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': ' ', 'version': '::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'apple', 'action': 'install', 'id': '5495053', 'version': '::fruits', 'url': ' '}, {'name': 'banana', 'action': 'install', 'id': '5494888', 'version': '::fruits', 'url': ' '}, {'name': 'asciijump', 'action': 'install', 'id': '5475369', 'version': '::apt', 'url': ' '}, {'name': 'cherry', 'action': 'install', 'id': '5495382', 'version': '::fruits', 'url': ' '}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '::docker', 'url': ' '}]}
10:06:15 INFO  State of operation 18298537 : PENDING
10:06:17 INFO  State of operation 18298537 : EXECUTING
10:06:19 INFO  State of operation 18298537 : EXECUTING
10:06:21 INFO  State of operation 18298537 : EXECUTING
10:06:23 INFO  State of operation 18298537 : EXECUTING
10:06:26 INFO  State of operation 18298537 : EXECUTING
10:06:29 INFO  State of operation 18298537 : EXECUTING
10:06:31 INFO  State of operation 18298537 : EXECUTING
10:06:34 INFO  State of operation 18298537 : EXECUTING
10:06:36 INFO  State of operation 18298537 : EXECUTING
10:06:38 INFO  State of operation 18298537 : EXECUTING
10:06:41 INFO  State of operation 18298537 : EXECUTING
10:06:43 INFO  State of operation 18298537 : EXECUTING
10:06:45 INFO  State of operation 18298537 : EXECUTING
10:06:47 INFO  State of operation 18298537 : EXECUTING
10:06:50 INFO  State of operation 18298537 : EXECUTING
10:06:52 INFO  State of operation 18298537 : FAILED
10:06:52 ERROR Stopping as the operation has failed
10:06:52 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1294, in __call__
    self.testObj.execute()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_mixed_install_remove_multiple/run.py", line 95, in execute
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
10:06:52 WARN  SystemError: The operation has failed ... blocked 
10:06:52 INFO  
10:06:52 INFO  cleanup:
10:06:56 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
10:06:56 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
10:06:56 INFO  
10:06:56 INFO  Test duration: 69.69 secs
10:06:56 INFO  Test final outcome:  BLOCKED
10:06:56 INFO  Test outcome reason: SystemError: The operation has failed (+1 other failures)
10:06:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="agent_init_session" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="AgentInitSession" name="agent_init_session">
		<failure message="FAILED: Grep on sub_resp.out contains &quot;Ld3KgqpcLDlrYH6sfpG7w&quot;" type="FAILED" />
		<system-out>09:03:29 INFO  ==============================================================
09:03:29 INFO  Id:    agent_init_session
09:03:29 INFO  Dir:   tedge_agent/
09:03:29 INFO  Title: Validate tedge agent init session feature
09:03:29 INFO  ==============================================================
09:03:29 INFO  Executed sudo&lt;remove_lock&gt;, exit status 0
09:03:30 INFO  Executed sudo&lt;agent_clear&gt;, exit status 0
09:03:30 INFO  Executed sudo&lt;agent_init&gt;, exit status 0
09:03:31 INFO  Executed sudo&lt;pub_req&gt;, exit status 0
09:03:31 INFO  Started sudo&lt;sub_resp&gt; with process id 16365
09:03:32 INFO  Executed sudo&lt;agent_start&gt;, exit status 0
09:03:32 INFO  Waiting for 2.0 seconds
09:03:34 INFO  Executed sudo&lt;agent_stop&gt;, exit status 0
09:03:35 INFO  Executed sudo&lt;sub_stop&gt;, exit status 0
09:03:35 WARN  Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w" ... failed [run.py:87]
09:03:35 INFO  
09:03:35 INFO  Test duration: 5.88 secs
09:03:35 INFO  Test final outcome:  FAILED
09:03:35 INFO  Test outcome reason: Grep on sub_resp.out contains "Ld3KgqpcLDlrYH6sfpG7w"
09:03:35 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="apt_fail_noarg" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFailNoArg" name="apt_fail_noarg">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_fail_noarg
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin fetures(prepare, list, install, remove finalize) without arguments
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.06 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFailNoArg" name="apt_fail_noarg" />
</testsuite><testsuite name="apt_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginFinalize" name="apt_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_finalize
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin finalize - dense version
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.04 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginFinalize" name="apt_finalize" />
</testsuite><testsuite name="apt_install" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallDense" name="apt_install">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin prepare - dense version
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.02 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallDense" name="apt_install" />
</testsuite><testsuite name="apt_install_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFail" name="apt_install_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install_fail
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin prepare - failing
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.03 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFail" name="apt_install_fail" />
</testsuite><testsuite name="apt_install_local_file" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install_local_file
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin install from local file - SUCCESS case
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.02 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFile" name="apt_install_local_file" />
</testsuite><testsuite name="apt_install_local_file_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install_local_file_fail
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin install from local file - FAIL case
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.02 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileFail" name="apt_install_local_file_fail" />
</testsuite><testsuite name="apt_install_local_file_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install_local_file_with_version
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin install from local file AND with a version - SUCCESS case
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.03 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersion" name="apt_install_local_file_with_version" />
</testsuite><testsuite name="apt_install_local_file_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:52:56 INFO  ==============================================================
08:52:56 INFO  Id:    apt_install_local_file_with_version_fail
08:52:56 INFO  Dir:   plugin_apt/
08:52:56 INFO  Title: Validate apt plugin install from local file AND with a version - FAIL case
08:52:56 INFO  ==============================================================
08:52:56 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:52:56 WARN  Aborted test due to SKIPPED outcome
08:52:56 INFO  
08:52:56 INFO  Test duration: 0.03 secs
08:52:56 INFO  Test final outcome:  SKIPPED
08:52:56 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:52:56 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallFromLocalFileWithVersionFail" name="apt_install_local_file_with_version_fail" />
</testsuite><testsuite name="apt_list_format" tests="2" errors="0" failures="0" skipped="0" time="0"><testcase classname="AptPluginListFormat" name="apt_list_format" />
<testcase classname="AptPluginListFormat" name="apt_list_format" />
</testsuite><testsuite name="apt_prepare" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepare" name="apt_prepare">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_prepare
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin install - dense version
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepare" name="apt_prepare" />
</testsuite><testsuite name="apt_prepare_install_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_prepare_install_finalize
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin install use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepareInstallFinalize" name="apt_prepare_install_finalize" />
</testsuite><testsuite name="apt_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveDense" name="apt_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_remove
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin prepare - dense version
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.04 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveDense" name="apt_remove" />
</testsuite><testsuite name="apt_remove_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveFails" name="apt_remove_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_remove_fail
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin prepare - failing
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveFails" name="apt_remove_fail" />
</testsuite><testsuite name="apt_sequence_install_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_sequence_install_with_version
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin install with version use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginPrepInstallWithVersionFinalize" name="apt_sequence_install_with_version" />
</testsuite><testsuite name="apt_sequence_install_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_sequence_install_with_version_fail
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin install with version fails use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallWithVersionFails" name="apt_sequence_install_with_version_fail" />
</testsuite><testsuite name="apt_sequence_remove" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_sequence_remove
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin prepare install remove finalize use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginInstallPrepareRemoveFinalize" name="apt_sequence_remove" />
</testsuite><testsuite name="apt_sequence_remove_with_version" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_sequence_remove_with_version
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin remove with version use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersion" name="apt_sequence_remove_with_version" />
</testsuite><testsuite name="apt_sequence_remove_with_version_fail" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_sequence_remove_with_version_fail
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin remove with version fails when the version not found
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginRemoveWithVersionFails" name="apt_sequence_remove_with_version_fail" />
</testsuite><testsuite name="apt_update_list" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginUpdateList" name="apt_update_list">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_update_list
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin update-list command use case
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginUpdateList" name="apt_update_list" />
</testsuite><testsuite name="apt_wrong_option" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="AptPluginWrongOption" name="apt_wrong_option">
		<skipped message="SKIPPED: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    apt_wrong_option
08:53:00 INFO  Dir:   plugin_apt/
08:53:00 INFO  Title: Validate apt plugin install
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the apt plugin is not supported on this platform.To run the tests call PySys with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="AptPluginWrongOption" name="apt_wrong_option" />
</testsuite><testsuite name="c8y_jwt_token_retrieval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="ValidateJWTTokenRetrieval" name="c8y_jwt_token_retrieval">
		<failure message="FAILED: Grep on resp_jwt.out contains &quot;\[c8y/s/dat\] 71,&quot;" type="FAILED" />
		<system-out>08:49:12 INFO  ==============================================================
08:49:12 INFO  Id:    c8y_jwt_token_retrieval
08:49:12 INFO  Dir:   cloud_c8y/
08:49:12 INFO  Title: Validate the Cumulocity JWT Token Retrieval feature
08:49:12 INFO  ==============================================================
08:49:13 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
08:49:27 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
08:49:27 INFO  Started sudo&lt;resp_jwt&gt; with process id 12372
08:49:29 INFO  Executed sudo&lt;req_jwt&gt;, exit status 0
08:49:31 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
08:49:31 WARN  Grep on resp_jwt.out contains "\[c8y/s/dat\] 71," ... failed [run.py:60]
08:49:31 INFO  
08:49:31 INFO  cleanup:
08:49:36 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
08:49:36 INFO  
08:49:36 INFO  Test duration: 23.74 secs
08:49:36 INFO  Test final outcome:  FAILED
08:49:36 INFO  Test outcome reason: Grep on resp_jwt.out contains "\[c8y/s/dat\] 71,"
08:49:36 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="c8y_restart_bridge" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="RestartBridge" name="c8y_restart_bridge" />
</testsuite><testsuite name="call_tedge_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-V" />
</testsuite><testsuite name="call_tedge_-h" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h" />
</testsuite><testsuite name="call_tedge_-h_-V" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_-h_-V" />
</testsuite><testsuite name="call_tedge_config_help" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_help" />
</testsuite><testsuite name="call_tedge_config_list" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="call_tedge_config_list" />
</testsuite><testsuite name="custom_init_system" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="CustomInitSystem" name="custom_init_system" />
</testsuite><testsuite name="docker_plugin_finalize" tests="2" errors="0" failures="0" skipped="1" time="0"><testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    docker_plugin_finalize
08:53:00 INFO  Dir:   plugin_docker/
08:53:00 INFO  Title: Validate docker plugin finalize command
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.05 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginFinalizeTest" name="docker_plugin_finalize" />
</testsuite><testsuite name="docker_plugin_install" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    docker_plugin_install
08:53:00 INFO  Dir:   plugin_docker/
08:53:00 INFO  Title: Validate docker plugin install
08:53:00 INFO  ==============================================================
08:53:00 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:00 WARN  Aborted test due to SKIPPED outcome
08:53:00 INFO  
08:53:00 INFO  Test duration: 0.02 secs
08:53:00 INFO  Test final outcome:  SKIPPED
08:53:00 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:00 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallTest" name="docker_plugin_install">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>09:44:39 INFO  ==============================================================
09:44:39 INFO  Id:    docker_plugin_install
09:44:39 INFO  Dir:   plugin_docker/
09:44:39 INFO  Title: Validate docker plugin install
09:44:39 INFO  ==============================================================
09:44:40 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
09:44:40 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_images.out ... passed
09:44:42 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
09:44:43 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:23]
09:44:43 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
09:44:44 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
09:44:44 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install/Output/linux/docker_ps.out contains "registry" ... failed [environment_docker_plugin.py:49,run.py:26]
09:44:44 INFO  
09:44:44 INFO  cleanup:
09:44:44 INFO  Removing containers scheduled to be cleaned up: ['']
09:44:45 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
09:44:45 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry']
09:44:46 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
09:44:46 INFO  
09:44:46 INFO  Test duration: 6.68 secs
09:44:46 INFO  Test final outcome:  BLOCKED
09:44:46 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
09:44:46 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_install_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:00 INFO  ==============================================================
08:53:00 INFO  Id:    docker_plugin_install_with_version
08:53:00 INFO  Dir:   plugin_docker/
08:53:00 INFO  Title: Validate docker plugin install
08:53:00 INFO  ==============================================================
08:53:01 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:01 WARN  Aborted test due to SKIPPED outcome
08:53:01 INFO  
08:53:01 INFO  Test duration: 0.02 secs
08:53:01 INFO  Test final outcome:  SKIPPED
08:53:01 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:01 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginInstallWithVersionTest" name="docker_plugin_install_with_version">
		<failure message="BLOCKED: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)" type="BLOCKED" />
		<system-out>09:44:46 INFO  ==============================================================
09:44:46 INFO  Id:    docker_plugin_install_with_version
09:44:46 INFO  Dir:   plugin_docker/
09:44:46 INFO  Title: Validate docker plugin install
09:44:46 INFO  ==============================================================
09:44:47 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
09:44:47 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_images.out ... passed
09:44:49 WARN  Executed sudo&lt;plugin_install&gt;, exit status 2
09:44:49 WARN  sudo&lt;plugin_install&gt; returned non-zero exit code 2 ... blocked [environment_docker_plugin.py:136,environment_docker_plugin.py:144,run.py:24]
09:44:50 INFO  Executed sudo&lt;docker_ps_latest&gt;, exit status 0
09:44:51 INFO  Executed sudo&lt;docker_ps&gt;, exit status 0
09:44:51 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_install_with_version/Output/linux/docker_ps.out contains "registry:2.7.1" ... failed [environment_docker_plugin.py:49,run.py:28]
09:44:51 INFO  
09:44:51 INFO  cleanup:
09:44:51 INFO  Removing containers scheduled to be cleaned up: ['', '']
09:44:52 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
09:44:52 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
09:44:53 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
09:44:53 INFO  
09:44:53 INFO  Test duration: 6.80 secs
09:44:53 INFO  Test final outcome:  BLOCKED
09:44:53 INFO  Test outcome reason: sudo&lt;plugin_install&gt; returned non-zero exit code 2 (+1 other failures)
09:44:53 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:01 INFO  ==============================================================
08:53:01 INFO  Id:    docker_plugin_remove
08:53:01 INFO  Dir:   plugin_docker/
08:53:01 INFO  Title: Validate docker plugin remove
08:53:01 INFO  ==============================================================
08:53:01 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:01 WARN  Aborted test due to SKIPPED outcome
08:53:01 INFO  
08:53:01 INFO  Test duration: 0.02 secs
08:53:01 INFO  Test final outcome:  SKIPPED
08:53:01 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:01 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>09:44:53 INFO  ==============================================================
09:44:53 INFO  Id:    docker_plugin_remove
09:44:53 INFO  Dir:   plugin_docker/
09:44:53 INFO  Title: Validate docker plugin remove
09:44:53 INFO  ==============================================================
09:44:54 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
09:44:54 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out ... passed
09:44:56 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
09:44:56 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:latest' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

09:44:56 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
09:44:56 WARN  Aborted test due to BLOCKED outcome
09:44:56 INFO  
09:44:56 INFO  cleanup:
09:44:56 INFO  Removing containers scheduled to be cleaned up: ['', '']
09:44:57 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
09:44:57 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
09:44:57 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
09:44:57 INFO  
09:44:57 INFO  Test duration: 4.75 secs
09:44:57 INFO  Test final outcome:  BLOCKED
09:44:57 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
09:44:57 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_remove_with_version" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:01 INFO  ==============================================================
08:53:01 INFO  Id:    docker_plugin_remove_with_version
08:53:01 INFO  Dir:   plugin_docker/
08:53:01 INFO  Title: Validate docker plugin remove with version
08:53:01 INFO  ==============================================================
08:53:01 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:01 WARN  Aborted test due to SKIPPED outcome
08:53:01 INFO  
08:53:01 INFO  Test duration: 0.02 secs
08:53:01 INFO  Test final outcome:  SKIPPED
08:53:01 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:01 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginRemoveWithVersionTest" name="docker_plugin_remove_with_version">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>09:44:57 INFO  ==============================================================
09:44:57 INFO  Id:    docker_plugin_remove_with_version
09:44:57 INFO  Dir:   plugin_docker/
09:44:57 INFO  Title: Validate docker plugin remove with version
09:44:57 INFO  ==============================================================
09:44:58 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
09:44:58 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove_with_version/Output/linux/docker_images.out ... passed
09:45:00 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
09:45:00 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.7.1' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

09:45:00 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
09:45:00 WARN  Aborted test due to BLOCKED outcome
09:45:00 INFO  
09:45:00 INFO  cleanup:
09:45:00 INFO  Removing containers scheduled to be cleaned up: ['', '']
09:45:01 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
09:45:01 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
09:45:02 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
09:45:02 INFO  
09:45:02 INFO  Test duration: 4.60 secs
09:45:02 INFO  Test final outcome:  BLOCKED
09:45:02 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
09:45:02 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="docker_plugin_update" tests="2" errors="0" failures="1" skipped="1" time="0"><testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<skipped message="SKIPPED: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'" />
		<system-out>08:53:01 INFO  ==============================================================
08:53:01 INFO  Id:    docker_plugin_update
08:53:01 INFO  Dir:   plugin_docker/
08:53:01 INFO  Title: Validate docker plugin install updates existing containers
08:53:01 INFO  ==============================================================
08:53:01 INFO  Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container' ... skipped
08:53:01 WARN  Aborted test due to SKIPPED outcome
08:53:01 INFO  
08:53:01 INFO  Test duration: 0.02 secs
08:53:01 INFO  Test final outcome:  SKIPPED
08:53:01 INFO  Test outcome reason: Testing the docker plugin is not supported on this platform.To run it, all the test with -XmyPlatform='container'
08:53:01 INFO  
</system-out>
	</testcase>
<testcase classname="DockerPluginUpdateTest" name="docker_plugin_update">
		<failure message="BLOCKED: sudo&lt;docker_run&gt; returned non-zero exit code 125" type="BLOCKED" />
		<system-out>09:45:02 INFO  ==============================================================
09:45:02 INFO  Id:    docker_plugin_update
09:45:02 INFO  Dir:   plugin_docker/
09:45:02 INFO  Title: Validate docker plugin install updates existing containers
09:45:02 INFO  ==============================================================
09:45:03 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
09:45:03 INFO  Grep on file /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_update/Output/linux/docker_images.out ... passed
09:45:05 WARN  Executed sudo&lt;docker_run&gt;, exit status 125
09:45:05 INFO  Contents of docker_run.err: 
  Unable to find image 'registry:2.6.2' locally
  docker: Error response from daemon: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit.
  See 'docker run --help'.
  -----

09:45:05 WARN  sudo&lt;docker_run&gt; returned non-zero exit code 125 ... blocked 
09:45:05 WARN  Aborted test due to BLOCKED outcome
09:45:05 INFO  
09:45:05 INFO  cleanup:
09:45:05 INFO  Removing containers scheduled to be cleaned up: ['', '']
09:45:06 WARN  Executed sudo&lt;containers_for_cleanup&gt;, exit status 1
09:45:06 INFO  Removing images scheduled to be cleaned up: ['registry', 'registry', 'registry:2.7.1']
09:45:07 INFO  Executed sudo&lt;containers_for_cleanup&gt;, exit status 0
09:45:07 INFO  
09:45:07 INFO  Test duration: 4.56 secs
09:45:07 INFO  Test final outcome:  BLOCKED
09:45:07 INFO  Test outcome reason: sudo&lt;docker_run&gt; returned non-zero exit code 125
09:45:07 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="invalid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="InvalidDeviceId" name="invalid_device_id" />
</testsuite><testsuite name="local_mqtt_pub_fail_noqos" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_noqos" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos0" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos0" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos1" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos1" />
</testsuite><testsuite name="local_mqtt_pub_fail_qos2" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_fail_qos2" />
</testsuite><testsuite name="local_mqtt_pub_sub" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub" />
</testsuite><testsuite name="local_mqtt_pub_sub_retain" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPublishWithRetain" name="local_mqtt_pub_sub_retain" />
</testsuite><testsuite name="local_mqtt_pub_sub_with_sudo" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_pub_sub_with_sudo" />
</testsuite><testsuite name="local_mqtt_sub_fail" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="local_mqtt_sub_fail" />
</testsuite><testsuite name="log_request_error_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifySearchTextError" name="log_request_error_messages">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>08:49:56 INFO  ==============================================================
08:49:56 INFO  Id:    log_request_error_messages
08:49:56 INFO  Dir:   log_request_end_to_end/
08:49:56 INFO  Title: Validate end to end behaviour for the log request operation check for error messages
08:49:56 INFO  ==============================================================
08:49:57 INFO  EnvironmentC8y Setup
08:49:58 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
08:49:58 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 2 seconds
08:49:58 INFO  Delaying execution by 9 seconds
08:50:21 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
08:50:22 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
08:50:23 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
08:50:23 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
08:50:24 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
08:50:24 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
08:50:25 INFO  log response None
08:50:25 WARN  retry, retrying in 1 seconds...
08:50:26 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18261381/binaries
08:50:26 WARN  Assert that (True == value) with value=False ... failed [run.py:43]
08:50:26 INFO  
08:50:26 INFO  cleanup:
08:50:27 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
08:50:27 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
08:50:28 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
08:50:32 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
08:50:33 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
08:50:33 INFO  
08:50:33 INFO  Test duration: 37.08 secs
08:50:33 INFO  Test final outcome:  FAILED
08:50:33 INFO  Test outcome reason: Assert that (True == value) with value=False
08:50:33 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="log_request_num_lines" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="LogRequestVerifyNumberOfLines" name="log_request_num_lines">
		<failure message="FAILED: Assert that (True == value) with value=False" type="FAILED" />
		<system-out>08:50:33 INFO  ==============================================================
08:50:33 INFO  Id:    log_request_num_lines
08:50:33 INFO  Dir:   log_request_end_to_end/
08:50:33 INFO  Title: Validate end to end behaviour for the log request operation check for number of lines
08:50:33 INFO  ==============================================================
08:50:33 INFO  EnvironmentC8y Setup
08:50:34 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
08:50:34 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
08:50:34 INFO  Delaying execution by 6 seconds
08:50:54 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
08:50:55 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
08:50:55 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
08:50:56 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
08:50:56 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
08:50:57 INFO  Executed sudo&lt;move_logs&gt;, exit status 0
08:50:57 INFO  log response None
08:50:57 WARN  retry, retrying in 1 seconds...
08:50:59 INFO  log response https://thin-edge-io.eu-latest.cumulocity.com/event/events/18270467/binaries
08:50:59 INFO  nlines 0
08:50:59 WARN  Assert that (True == value) with value=False ... failed [run.py:44]
08:50:59 INFO  
08:50:59 INFO  cleanup:
08:50:59 INFO  Executed sudo&lt;rm_logs&gt;, exit status 0
08:51:00 INFO  Executed sudo&lt;rm_tmp_logs&gt;, exit status 0
08:51:01 INFO  Executed sudo&lt;send_failed&gt;, exit status 0
08:51:05 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
08:51:05 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
08:51:05 INFO  
08:51:05 INFO  Test duration: 32.26 secs
08:51:05 INFO  Test final outcome:  FAILED
08:51:05 INFO  Test outcome reason: Assert that (True == value) with value=False
08:51:05 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mapper_awaits_before_reconnect" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MapperReconnectAwait" name="mapper_awaits_before_reconnect">
		<failure message="FAILED: Line count on tedge_mapper_journal.out for &quot;tedge_mapper::core::mapper: MQTT connection error:&quot; expected &gt;=2 but got 0" type="FAILED" />
		<system-out>09:05:40 INFO  ==============================================================
09:05:40 INFO  Id:    mapper_awaits_before_reconnect
09:05:40 INFO  Dir:   tedge_mapper_c8y/
09:05:40 INFO  Title: Validate mapper doesn't reconnect too often
09:05:40 INFO  ==============================================================
09:05:40 INFO  Started mosquitto_sub with process id 17123
09:05:40 INFO  Waiting for 0.1 seconds
09:05:41 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
09:05:41 INFO  Waiting for 5.0 seconds
09:05:46 INFO  Executed sudo&lt;tedge_mapper&gt;, exit status 0
09:05:47 INFO  Executed sudo&lt;tedge_mapper_journal&gt;, exit status 0
09:05:47 WARN  Grep on tedge_mapper_journal.out contains "ERROR" ... failed [run.py:67]
09:05:47 INFO  Line count on file tedge_mapper_journal.out ... passed
09:05:47 WARN  Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0 ... failed [run.py:78]
09:05:47 WARN  Aborted test due to FAILED outcome
09:05:48 INFO  
09:05:48 INFO  Test duration: 7.40 secs
09:05:48 INFO  Test final outcome:  FAILED
09:05:48 INFO  Test outcome reason: Line count on tedge_mapper_journal.out for "tedge_mapper::core::mapper: MQTT connection error:" expected &gt;=2 but got 0
09:05:48 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_smaller_interval" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringSmallInterval" name="monitoring_smaller_interval">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>08:52:36 INFO  ==============================================================
08:52:36 INFO  Id:    monitoring_smaller_interval
08:52:36 INFO  Dir:   monitoring/
08:52:36 INFO  Title: Validate the tedge dm agent messages that are produced with smaller sampling interval than the batching window. Here the simulated collectd messages are used
08:52:36 INFO  ==============================================================
08:52:37 INFO  Executed sudo&lt;collectd&gt;, exit status 0
08:52:37 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
08:52:37 INFO  Started sudo&lt;tedge_sub&gt; with process id 13290
08:52:38 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:38 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:39 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:39 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:40 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:41 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:41 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:42 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:42 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:43 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:43 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:44 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:45 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:45 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:46 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:46 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:47 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:47 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:48 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:48 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:50 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
08:52:50 INFO  0

08:52:50 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 104, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 114, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_smaller_interval/run.py", line 134, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
08:52:50 WARN  TypeError: 'int' object is not subscriptable ... blocked 
08:52:50 INFO  
08:52:50 INFO  cleanup:
08:52:50 INFO  monitoring_cleanup
08:52:51 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
08:52:51 INFO  
08:52:51 INFO  Test duration: 14.97 secs
08:52:51 INFO  Test final outcome:  BLOCKED
08:52:51 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
08:52:51 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_collectd" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MonitoringWithCollectd" name="monitoring_with_collectd">
		<skipped message="SKIPPED: Test execution disabled due to #792 (instable)" />
		<system-out>08:52:51 INFO  ==============================================================
08:52:51 INFO  Id:    monitoring_with_collectd
08:52:51 INFO  Dir:   monitoring/
08:52:51 INFO  Title: Validate tedge dm agent messages that are produced by batching the collectd messages
08:52:51 INFO  ==============================================================
08:52:51 INFO  Test execution disabled due to #792 (instable) ... skipped
08:52:51 INFO  
08:52:51 INFO  Test duration: 0.02 secs
08:52:51 INFO  Test final outcome:  SKIPPED
08:52:51 INFO  Test outcome reason: Test execution disabled due to #792 (instable)
08:52:51 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="monitoring_with_simulated_messages" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="MonitoringWithSimulatedMessages" name="monitoring_with_simulated_messages">
		<failure message="BLOCKED: TypeError: 'int' object is not subscriptable" type="BLOCKED" />
		<system-out>08:52:51 INFO  ==============================================================
08:52:51 INFO  Id:    monitoring_with_simulated_messages
08:52:51 INFO  Dir:   monitoring/
08:52:51 INFO  Title: Validate the tedge dm agent messages that are produced from the simulated collectd messages
08:52:51 INFO  ==============================================================
08:52:51 INFO  Executed sudo&lt;collectd&gt;, exit status 0
08:52:52 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
08:52:52 INFO  Started sudo&lt;tedge_sub&gt; with process id 13343
08:52:53 INFO  Executed sudo&lt;tedge_temp&gt;, exit status 0
08:52:53 INFO  Executed sudo&lt;tedge_pres&gt;, exit status 0
08:52:55 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
08:52:55 INFO  0

08:52:55 WARN  TypeError occurred while running test - 'int' object is not subscriptable
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1296, in __call__
    self.testObj.validate()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 98, in validate
    collectd_msg_validation_result=self.validate_json(),
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 108, in validate_json
    if not self.validate_time():
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/monitoring/monitoring_with_simulated_messages/run.py", line 127, in validate_time
    if self.js_msg["time"]:
TypeError: 'int' object is not subscriptable
08:52:55 WARN  TypeError: 'int' object is not subscriptable ... blocked 
08:52:55 INFO  
08:52:55 INFO  cleanup:
08:52:55 INFO  monitoring_cleanup
08:52:56 INFO  Executed sudo&lt;collectd_mapper&gt;, exit status 0
08:52:56 INFO  
08:52:56 INFO  Test duration: 4.69 secs
08:52:56 INFO  Test final outcome:  BLOCKED
08:52:56 INFO  Test outcome reason: TypeError: 'int' object is not subscriptable
08:52:56 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_change_connection_fails" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortChangeConnectionFails" name="mqtt_port_change_connection_fails" />
</testsuite><testsuite name="mqtt_port_change_connection_works" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="MqttPortChangeConnectionWorks" name="mqtt_port_change_connection_works">
		<skipped message="SKIPPED: failing intermitently" />
		<system-out>08:51:41 INFO  ==============================================================
08:51:41 INFO  Id:    mqtt_port_change_connection_works
08:51:41 INFO  Dir:   misc_features/
08:51:41 INFO  Title: mqtt port change connection works for all the thin-edge services that use the mosquitto broker
08:51:41 INFO  ==============================================================
08:51:41 INFO  failing intermitently ... skipped
08:51:41 INFO  
08:51:41 INFO  Test duration: 0.02 secs
08:51:41 INFO  Test final outcome:  SKIPPED
08:51:41 INFO  Test outcome reason: failing intermitently
08:51:41 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="mqtt_port_set" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortSet" name="mqtt_port_set" />
</testsuite><testsuite name="mqtt_port_unset" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="MqttPortUnSet" name="mqtt_port_unset" />
</testsuite><testsuite name="publish_flux" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishFlux" name="publish_flux" />
</testsuite><testsuite name="publish_sawmill" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PublishSawmill" name="publish_sawmill" />
</testsuite><testsuite name="runtime_start_two_mappers" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="RuntimeMultiMappers" name="runtime_start_two_mappers">
		<failure message="FAILED: Grep on tedge_mapper2.err contains &quot;Error: Couldn't acquire file lock.&quot;" type="FAILED" />
		<system-out>09:05:48 INFO  ==============================================================
09:05:48 INFO  Id:    runtime_start_two_mappers
09:05:48 INFO  Dir:   tedge_mapper_c8y/
09:05:48 INFO  Title: Validate two mappers can't be started at the same time
09:05:48 INFO  ==============================================================
09:05:48 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
09:05:48 INFO  Waiting for 0.1 seconds
09:05:49 INFO  Executed sudo&lt;tedge_mapper2&gt;, exit status 1
09:05:49 INFO  Executed sudo&lt;tedge_mapper1&gt;, exit status 0
09:05:49 WARN  Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock." ... failed [run.py:46]
09:05:49 INFO  
09:05:49 INFO  Test duration: 1.76 secs
09:05:49 INFO  Test final outcome:  FAILED
09:05:49 INFO  Test outcome reason: Grep on tedge_mapper2.err contains "Error: Couldn't acquire file lock."
09:05:49 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="sm_tedge_mapper_init_session" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeSMMapperInitSession" name="sm_tedge_mapper_init_session" />
</testsuite><testsuite name="smoketest_json" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmokeTestJson" name="smoketest_json" />
</testsuite><testsuite name="smoketest_json_400_samples_10ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples10ms" name="smoketest_json_400_samples_10ms" />
</testsuite><testsuite name="smoketest_json_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestJson400Samples20ms" name="smoketest_json_400_samples_20ms" />
</testsuite><testsuite name="smoketest_smartrest" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest" name="smoketest_smartrest" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_1Xms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples1Xms" name="smoketest_smartrest_400_samples_1Xms" />
</testsuite><testsuite name="smoketest_smartrest_400_samples_20ms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="SmoketestSmartRest400Samples20ms" name="smoketest_smartrest_400_samples_20ms" />
</testsuite><testsuite name="tedge_agent_user_sudo_access" tests="1" errors="0" failures="0" skipped="1" time="0"><testcase classname="TedgeAgentUserSudoAccess" name="tedge_agent_user_sudo_access">
		<skipped message="SKIPPED: #955 Observation: Exit code of the apt plugin when called with no arguments" />
		<system-out>09:03:03 INFO  ==============================================================
09:03:03 INFO  Id:    tedge_agent_user_sudo_access
09:03:03 INFO  Dir:   tedge/
09:03:03 INFO  Title: Validate tedge-agent user has a limited sudo right
09:03:03 INFO  ==============================================================
09:03:03 INFO  #955 Observation: Exit code of the apt plugin when called with no arguments ... skipped
09:03:03 INFO  
09:03:03 INFO  Test duration: 0.02 secs
09:03:03 INFO  Test final outcome:  SKIPPED
09:03:03 INFO  Test outcome reason: #955 Observation: Exit code of the apt plugin when called with no arguments
09:03:03 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_connect_test_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestNegative" name="tedge_connect_test_negative" />
</testsuite><testsuite name="tedge_connect_test_positive" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestPositive" name="tedge_connect_test_positive" />
</testsuite><testsuite name="tedge_connect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeConnectTestSMServices" name="tedge_connect_test_sm_services" />
</testsuite><testsuite name="tedge_disconnect_test_sm_services" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeDisConnectTestSMServices" name="tedge_disconnect_test_sm_services" />
</testsuite><testsuite name="tedge_mapper_az_negative" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzError" name="tedge_mapper_az_negative" />
</testsuite><testsuite name="tedge_mapper_az_with_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithTimestamp" name="tedge_mapper_az_with_timestamp" />
</testsuite><testsuite name="tedge_mapper_az_without_timestamp" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperAzWithoutTimestamp" name="tedge_mapper_az_without_timestamp" />
</testsuite><testsuite name="tedge_mapper_c8y_alarms" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yAlarm" name="tedge_mapper_c8y_alarms" />
</testsuite><testsuite name="tedge_mapper_c8y_child_device_measurement" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="TedgeMapperC8yChildDevice" name="tedge_mapper_c8y_child_device_measurement" />
</testsuite><testsuite name="tedge_mapper_c8y_packet_threshold_size" tests="1" errors="0" failures="1" skipped="0" time="0"><testcase classname="TedgeMapperC8yThresholdPacketSize" name="tedge_mapper_c8y_packet_threshold_size">
		<failure message="FAILED: Grep on tedge_sub.out contains &quot;The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184.&quot;" type="FAILED" />
		<system-out>09:07:24 INFO  ==============================================================
09:07:24 INFO  Id:    tedge_mapper_c8y_packet_threshold_size
09:07:24 INFO  Dir:   tedge_mapper_c8y/
09:07:24 INFO  Title: Validate the Cumulocity Mapper accepted threshold packet size when it receives a message on tedge/measurements
09:07:24 INFO  ==============================================================
09:07:25 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
09:07:25 INFO  Started sudo&lt;tedge_sub&gt; with process id 17715
09:07:25 INFO  Executed sudo&lt;tedge_msg&gt;, exit status 0
09:07:26 INFO  Executed sudo&lt;tedge_pub&gt;, exit status 0
09:07:27 INFO  Executed sudo&lt;kill_out&gt;, exit status 0
09:07:27 WARN  Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184." ... failed [run.py:88]
09:07:27 INFO  
09:07:27 INFO  cleanup:
09:07:27 INFO  mapper_cleanup
09:07:27 INFO  Executed sudo&lt;tedge_mapper_c8y&gt;, exit status 0
09:07:27 INFO  
09:07:27 INFO  Test duration: 3.06 secs
09:07:27 INFO  Test final outcome:  FAILED
09:07:27 INFO  Test outcome reason: Grep on tedge_sub.out contains "The size of the message received on tedge/measurements is 20489 which is greater than the threshold size of 16184."
09:07:27 INFO  
</system-out>
	</testcase>
</testsuite><testsuite name="tedge_software_update" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="PySysTest" name="tedge_software_update" />
</testsuite><testsuite name="valid_device_id" tests="1" errors="0" failures="0" skipped="0" time="0"><testcase classname="ValidateValidDeviceId" name="valid_device_id" />
</testsuite></testsuites>